servatrice/src/servatrice_database_interface.cpp

```diff
@@ -175,10 +175,10 @@ bool Servatrice_DatabaseInterface::registerUser(const QString &userName, const Q
     QString passwordSha512 = PasswordHasher::computeHash(password, PasswordHasher::generateRandomSalt());
     token = active ? QString() : PasswordHasher::generateActivationToken();
 
-    QSqlQuery *query = prepareQuery("insert into {prefix}_users "
-            "(name, realname, gender, password_sha512, email, country, registrationDate, active, token) "
-            "values "
-            "(:userName, :realName, :gender, :password_sha512, :email, :country, UTC_TIMESTAMP(), :active, :token)");
+	QSqlQuery *query = prepareQuery("insert into {prefix}_users "
+		"(name, realname, gender, password_sha512, email, country, registrationDate, active, token, admin, avatar_bmp, clientid, privlevel, privlevelStartDate, privlevelEndDate) "
```

Comment:
Yes, that is what this was for.  I can't say I looked else were the strict mode may be affecting but this does resolve the issue when user account information is added in strict mode.